.github/workflows/build.yml

```diff
@@ -270,6 +279,42 @@ jobs:
           rm -rf temp
           mv signed temp
 
+      - name: Generate VSA
+        if: env.shouldSign == 'true'
+        env:
+          AWS_KMS_REGION:  ${{ job.build.steps.auth.with.args.aws-region }}
+          AWS_KMS_KEY_ARN: ${{ job.build.steps.sign.env.AWS_KMS_KEY_ARN }}
+        run: |
+          args+=(
+            --mount "type=bind,src=$PWD/signed,dst=/doi-build/signed"
+
+            # https://explore.ggcr.dev/?repo=docker/image-signer-verifier
+            "docker/image-signer-verifier:$BASHBREW_IMAGE_SIGNER"
+
+            verify
+
+            -D
+
+            --platform="$BASHBREW_ARCH"
+            --vsa=true
+
+            --tuf-source=oci
+            --tuf-meta-source="$BASHBREW_TUF_META"
+            --tuf-targets-source="$BASHBREW_TUF_TARGETS"
+
+            --policy-id="$BASHBREW_SIGNING_POLICY"
```

Comment:
For example, is this referencing some kind of written policy inside `image-signer-verifier` itself?  A codified (rego, etc) policy?  Where could I find more details about what the possible arguments to this flag are and what they mean / what we're attesting by using them?

Reply:
Yes, I think comments for these input args would go a long way. I'll help with that